### ISSUE TITLE
Jupyter - Nothing happens 

### ISSUE THREAD

**Issue by Jacques2101** [streetsidesoftware/vscode-spell-checker#1247]
Hi, I am sorry but I am new in Vscode/Spell and after installing it I see nothing happened and I have some questions 

1) does it work inside Jupyter ? 
2) I tried to type 
`# costant` which should generate an error (as in your example) but nothing happens 
3) does it work for markdown text in jupyter file ? or python ? 

Maybe it's my configuration that is a problem ???
Thx for your help 
<img width="1624" alt="Capture d’écran 2021-09-01 à 11 24 00" src="https://user-images.githubusercontent.com/49912455/131647254-db07b214-f7ba-4e02-90a6-fbe920f51825.png">
<img width="1624" alt="Capture d’écran 2021-09-01 à 11 24 29" src="https://user-images.githubusercontent.com/49912455/131647260-3812c986-6f55-4bfd-96cf-64aa30a7a90f.png">

**Comment by Jason3S** [comment]
Possible duplicate of #420

**Comment by Jacques2101** [comment]
OK so I am not sure that it works with jupyter (#420) but I have the same problem with Python file. 
I tried: 
`# costant`wich should generate an error but nothing happens where as I expected a squiggly underline (cf. the second screenshot I sent to you : the first is jupyter screen and the second python file)

**Comment by Jason3S** [comment]
@Jacques2101 
That is a different issue related to `allowCompoundWords`.

To see where the spell checker finds the word try:
`npx cspell trace --languageId=python costant`

![image](https://user-images.githubusercontent.com/3740137/131805063-f89d2f58-f2b8-42e3-bab6-0e83f06be962.png)

Because `allowCompoundWords` is turned on for Python, the spell checker looks for words to glue together. See #683 on how to turn it off.

**Comment by Jacques2101** [comment]
It seems to work now but also for instruction like Matplotlib...(cf screenshot). 

<img width="1274" alt="Capture d’écran 2021-09-02 à 16 07 56" src="https://user-images.githubusercontent.com/49912455/131858778-529e3be4-3b4d-4d63-855f-f5af8eab680f.png">

**Comment by Jason3S** [comment]
It looks like the Python library is missing some common imports.

In any case you can do this:

```
# cspell:includeRegExp comments
```

![image](https://user-images.githubusercontent.com/3740137/131872855-601aa5f6-4292-4104-aa61-494d9dd2cadd.png)

**Comment by Jacques2101** [comment]
ok but not sure what I have to do precisely. 
`# cspell:includeRegExp comments` where ?

**Comment by Jason3S** [comment]
As a comment in your code, like the example image:
![image](https://user-images.githubusercontent.com/3740137/131901051-fbd2d8b0-4b3e-40e9-b47b-330f8bde92e4.png)


Generally most people spell check the entire document, not just the comments.

Add this file to your project to just check comments in Python files:

`cspell.json`
```js
{
    "languageSettings": [
        {
            "languageId": "python",
            "includeRegExpList": [
                "comments"
            ]
        }
    ]
}
```

**Comment by Jacques2101** [comment]
Ok it seems to work now. I have correction in jupyter / python file. Just for comment and markdown cells. 

Where to learn how to configure spell ? not clear where to find these informations. 

thx. 

I did: 
```
"cSpell.languageSettings": [
        {
            "languageId": "python",
            "allowCompoundWords": false,
            "includeRegExpList": [
                "comments"
            ]
        }
```

**Comment by Jason3S** [comment]
Documentation is a bit of a work in progress:
- [Spell Checker - Spell Checker](https://streetsidesoftware.github.io/vscode-spell-checker/)
- [Language Settings - CSpell](https://streetsidesoftware.github.io/cspell/configuration/language-settings/#LanguageSettings)

**Comment by Jason3S** [comment]
I believe this is now working. Please open a new issue if it is not and refer to this one.
